src/build.nix

```diff
@@ -151,6 +151,10 @@ let
     {
       inherit pkgs release;
       runTests = doCheck;
+      testCrateFlags = [];
+      testInputs = [];
+      testPreRun = "";
+      testPostRun = "";
```

Comment:
Do these need to be specified here? I think it will work fine if you remove these but keep others.

Reply:
I tried without and it complained with errors and failed to build, but I didn't investigate further

Reply:
Well, the tests pass fine, so it should be OK. Will merge it now.